app/controllers/api/base_controller.rb

```diff
@@ -26,5 +26,30 @@ def valid_authentication_token?
       # TODO: Replace this with a proper authentication check
       request.headers["Authorization"] == "Bearer bat"
     end
+
+    def provider
```

Comment:
Would it make sense to call this `current_provider` to be consistent with other 'authenticated thing' methods?